pkg/specgen/container_validate.go

```diff
@@ -68,18 +68,6 @@ func (s *SpecGenerator) Validate() error {
 	if len(s.CapAdd) > 0 && s.Privileged {
 		return exclusiveOptions("CapAdd", "privileged")
 	}
-	// selinuxprocesslabel and privileged are exclusive
```

Comment:
Why are we turning these off? Privileged flag disables all manual SELinux config, last I checked

Reply:
In the other code, we are setting the selinux options to "disabled" when --privileged mode is enabled.  I guess I could check over there.  This was causing an error to be thrown here.